sphinxdocs/private/sphinx.bzl

```
@@ -177,6 +176,9 @@ def sphinx_docs(
         **common_kwargs
     )
 
+    common_kwargs_with_manual_tag = dict(common_kwargs)
+    common_kwargs_with_manual_tag.setdefault("tags", []).append("manual")
```

The dict() operation is shallow, so the tags object in `_with_manual` will refer to the same object as `common_kwargs`, and this will end up appending to both.

```suggestion
    common_kwargs_with_manual_tag["tags"] = list(common_kwargs.get("tags") or []) + ["manual"]
```